src/Runner.Worker/Container/ContainerHooks/ContainerHookManager.cs

```diff
@@ -193,8 +193,9 @@ private async Task<HookResponse> ExecuteHookScript(IExecutionContext context, Ho
             await handler.RunAsync(stage);
             if (handler.ExecutionContext.Result == TaskResult.Failed)
             {
-                throw new Exception("Hook failed"); // TODO: better exception
+                throw new Exception($"Hook command '{input.Command}' failed"); // TODO: better exception
```

Comment:
Maybe including the path could be useful? Something like `$"The hook script at '{filePath}' running command '{input.Command}'  did not execute successfully."`

Reply:
Let's add path to both logging objects